src/Login.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import axios from "axios";
+import history from './history';
+import './todo.css';
+
+class Home extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            username: "",
+            password: "",
+            cred: {}
+        };
+    }
+
+    setUsername = event => {
+        let value = event.target.value;
+        this.setState({
+            username: value
+        });
+    };
+
+    setPassword = event => {
+        let value = event.target.value;
+        this.setState({
+            password: value
+        });
+    };
+
+    logIn = () => {
+        if (this.state.username === '' || this.state.password === '') {
```

Comment:
you can validate onClick of input tag, rather than setting it to state then calling logIn function